### PR TITLE
used information from configuration file

### PR DESCRIPTION
**Pull request by ganeshpatro321**
I have made the necessary changes regarding issue #367 .
I couldn't check the working as I was having a problem with getting demo data.

Executing the jar file is showing this problem:
screenshot from 2019-02-23 00-09-59

### CODE REVIEW

**Comment by ganeshpatro321** [comment]
@jingchunzhu Please review this.

File: js/views/Column.js [hunk 1]
```diff
@@ -256,30 +268,10 @@ function supportsTumorMap({fieldType, fields, cohort, fieldSpecs}) {
...
-			'TCGA Pan-Cancer (PANCAN)': {
-				map: "PancanAtlas/SampleMap",
-				layout: "mRNA"
-			},
-			'GDC Pan-Cancer (PANCAN)': {
-				map: "xena_test/remapped_pancan_mrna",
-				layout: "layout"
-			}
-		};
+	var tumorMapLinkoutURL = `${cohortMetaData}/defaultTumormap.json`;
```
[the remark this answers is not in the document]
Comment (ganeshpatro321) [reply js/views/Column.js]: Sorry for the misconception, so you want me to get the data in the xenaquery.js and import that data to the columns page?

File: js/views/Column.js [hunk 2]
```diff
@@ -27,6 +27,10 @@ import { matches } from 'static-interval-tree';
 var gaEvents = require('../gaEvents');
 var crosshair = require('./cursor.png');
 
+var Rx = require('../rx');
+import cohortMetaData from '../cohortMetaData';
+
+
 var ESCAPE = 27;
```
[the remark this answers is not in the document]
Comment (ganeshpatro321) [reply js/views/Column.js]: I haven't added this code,

however this ESCAPE variable is used for defining the keycode of esc button, and the functionality is whenever esc key is clicked the menu is hidden in the app.

File: js/views/Column.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jingchunzhu) [reply js/views/Column.js]: Yes. So the configuration of all the metadata files is at the same place.

File: js/views/Column.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jingchunzhu) [reply js/views/Column.js]: why do you think there is a need to define the keycode of esc button?

**Pull request closed without merging**